scripts/convex.blocknative.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from 'node-fetch'
+
+// Add your keys via .env
+const credentials = process.env.BN_API_KEY + ':' + process.env.BN_SECRET_KEY
+const bnEndPoint = 'https://api.blocknative.com/simulate'
+
+const gasSupplied = 300000000 // Example high gas to succeed EVM execution, **currently no estimation**
+
+async function main(): Promise<void> {
+
+  // Double check the keys were given above!
+  if (credentials === ':') {
+    throw new Error('No credentials for simulation endpoint given!')
+  }
+
+  // Information to how tx-preview works is here:
+  // https://docs.blocknative.com/simulation-platform/transaction-preview-api
+  const body = {
+    system: "ethereum",
+    network: "main",
+    transaction: {
```

Comment:
Is there any way to specify the a block number to simulate at? Right now we've been simulating at block 13724056. If not, no worries and we can just document that difference in the README + `console.log()` it in this file so we can easily benchmark other tools at the same block

Reply:
Currently we do not run simulations on historical nodes as we have been focusing on speed for current blockheight simulations. We do plan to do this in the future! For now I added the console.logs() to describe this.